hack/generate/samples/internal/go/v2/memcached_with_webhooks.go

```diff
@@ -417,6 +419,7 @@ const importsFragment = `
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"reflect"
```

Comment:
Nit: can you move std packages to the top of the import block? Ex.
```go
import (
	"reflect"
	"time"

	appsv1 "k8s.io/api/apps/v1"
	...
)
```
Can also be done in a follow-up